pandas/tests/indexes/timedeltas/test_timedelta.py

```diff
@@ -509,6 +509,10 @@ def test_fields(self):
         tm.assert_series_equal(s.dt.seconds, Series(
             [10 * 3600 + 11 * 60 + 12, np.nan], index=[0, 1]))
 
+        # preserve name (GH15589)
```

Comment:
yeah prob should just define these as fixtures I think, then would make it really easy

https://github.com/pandas-dev/pandas/blob/master/pandas/tests/series/test_datetime_values.py#L30